plugins/bazaar/src/components/CardContentFields/CardContentFields.tsx

```diff
@@ -111,7 +113,14 @@ export const CardContentFields = ({
                         />
                         <Link
                           className={classes.break}
-                          to={`http://github.com/${member.userId}`}
+                          to={
+                            member.userRef
+                              ? generatePath(
+                                  '/catalog/:namespace/:kind/:name',
+                                  parseEntityRef(member.userRef),
+                                )
```

Comment:
Use `useRouteRef(entityRouteRef)` for generating the link instead. It supports other locations of the catalog entities as well as other app base paths.